# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../img2pdf/fix_make_static.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-03-06 11:32:41.000000000 -0300
+++ ./Makefile	2007-03-06 11:35:35.000000000 -0300
@@ -8,12 +8,13 @@
 	  -L/usr/local/lib -lpanda -lpng -ltiff -ljpeg -lz -lm
 
 img2pdf-static:	img2pdf.o
-	$(LD) -o $@ img2pdf.o \
-	  /usr/local/lib/libpanda.a \
-	  /usr/local/lib/libpng.a \
-	  /usr/local/lib/libtiff.a \
-	  /usr/local/lib/libjpeg.a \
-	  /usr/local/lib/libz.a -lm
+	$(LD) -o $@ img2pdf.o -static \
+	  -lpanda \
+	  -lpng \
+	  -ltiff \
+	  -ljpeg \
+	  -lz -lm -ldb \
+	  -lpthread
 
 img2pdf.o:	img2pdf.c
 	$(CC) -g -O2 -Wall -I/usr/local/include -c img2pdf.c
